src/to-govspeak.js

```diff
@@ -100,8 +115,24 @@ function removeBrParagraphs (govspeak) {
   return govspeak.replace(regExp, '\n')
 }
 
+function cleanUpNestedLinks (govspeak) {
+  // This finds instances of nested links and replaces them with the captured
+  // second occurrence of the URL.
+  const nestedLinkRegExp = new RegExp(/]\(\[.+?(?=])(](\().+?\))\)/, 'g')
```

Comment:
Nice one with the regex! I'm not sure I fully understand what it's doing :-|. Since we're expecting the link to come right after the '](', would it possible to do this with turndown, where we could just check 'end of preceding node is "]("' && 'current node is <a>'?

Reply:
Yeah I appreciate the regex is pretty impenetrable!  I'll see if I can make it work with what you've suggested.